### ISSUE TITLE
[Accessibility] The contrast ratio of text with its background must meet WCAG 2.1 AA requirements

### ISSUE THREAD

**Issue by PaigeChen** [carbon-design-system/carbon#8741]
## Environment

> Operating system

Mac OS Big Sur 11.3

> Browser

Chrome v90.0.4430.212

> Automated testing tool and ruleset

IBM Equal Access Accessibility Checker (27 April 2021 - Latest Deployment)

## Detailed description

> What version of the Carbon Design System are you using?

carbon-components v10.35
carbon-components-react v7.35

> What did you expect to happen?

No violations from the IBM Equal Access Accessibility Checker scan.

> What happened instead?

Violation "Text contrast of 1.71 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400" detected.

> What WCAG 2.1 checkpoint does the issue violate?

1.4.3 Contrast (Minimum) 

## Steps to reproduce the issue
1. Bring up product UI panel
2. Select "Storage volumes" in the left navigator
3. Click "Create volume" button
4. Run A11y Checker tool against this page. (Using "IBM Accessibility 27 April 2021 ruleset")

> Please create a reduced test case in CodeSandbox

https://codesandbox.io/s/wonderful-bassi-q7ymp?file=/index.js
https://q7ymp.csb.app/

## Additional information

- Screenshots or code
Below is the screenshot of my product where has the issue.
![image](https://user-images.githubusercontent.com/54969794/119316386-7bf3c400-bca9-11eb-90c1-6c4619e38191.png)

Below is the screenshot of the CodeSandbox that created the same issue as my product.
![image](https://user-images.githubusercontent.com/54969794/119316068-17386980-bca9-11eb-8ef5-8ec4716f7d08.png)

- Notes

**Comment by tw15egan** [comment]
Hey there! This actually seems to be a false positive from the a11y checker, as disabled items do not need to meet contrast requirements. Since they are inaccessible for keyboard/screenreader users, no information should be conveyed via a disabled item. 

https://www.w3.org/TR/WCAG21/#contrast-minimum

Specifically, 
>Incidental: Text or images of text that are part of an **inactive user interface component**

**Comment by PaigeChen** [comment]
Thanks for the prompt reply! I did some manual checks and find it's actually a hidden element. Should be an a11y checker issue. Thanks.
